manim/mobject/types/image_mobject.py

```diff
@@ -93,7 +93,8 @@ def __init__(
         image_mode="RGBA",
         **kwargs,
     ):
-        self.opacity = 0
+        self.fill_opacity = 1
```

Comment:
Can you please explain how the changes in this file relate to passing the background color to the renderer?

Reply:
Those changes are actually part of a different change, but I thought submitting a new PR would be too slow. I'll add them to the description.